### PR TITLE
feat: integrate superteam bounties

### PR DESCRIPTION
**Pull request by srijanshetty**
Integrate Superteam Bounties

### CODE REVIEW

File: hub/components/Hub/SideCard/Bounties/index.tsx [hunk 1]
```diff
@@ -1,7 +1,124 @@
...
+          'Accept': 'application/json',
+          'Content-Type': 'application/json',
+        },
+      });
+      const json = await res.json();
+
+      if (res.ok) {
+        try {
+          const listing = JSON.parse(json.data.value);
+          console.log(listing);
```
Comment (nramadas) [inline hub/components/Hub/SideCard/Bounties/index.tsx:42]: could we remove the console.log?

File: hub/components/Hub/SideCard/Bounties/index.tsx [hunk 2]
```diff
@@ -1,7 +1,124 @@
...
+              'text-left',
+              'tracking-normal',
+              'w-full',
+            )}
+            key={bounty.objectID}
+            onClick={() => {
+              const url = `https://earn.superteam.fun/${bounty.type.toLowerCase()}/${
+                bounty.objectID
+              }`;
+              router.push(url);
```
Comment (nramadas) [inline hub/components/Hub/SideCard/Bounties/index.tsx:77]: could we open in a new tab instead?

**Review by nramadas — CHANGES_REQUESTED** [review]
Looks great! just a few small nits.

Also, what's a good way to test this? Is there a realm that has bounties I could take a look at?

File: hub/components/Hub/SideCard/Bounties/index.tsx (hunk 1 above)
Comment (srijanshetty) [reply hub/components/Hub/SideCard/Bounties/index.tsx:42]: ORCA

Comment (srijanshetty) [reply hub/components/Hub/SideCard/Bounties/index.tsx:42]: The console.log should not be there, will clean in up. 

Will also share the company with a listing and add a screenshot for refernce

File: hub/components/Hub/SideCard/Bounties/index.tsx (hunk 2 above)
Comment (srijanshetty) [reply hub/components/Hub/SideCard/Bounties/index.tsx:77]: Will do ser!

**Comment by nramadas** [comment]
@srijanshetty hey! Would you mind giving me a test realm I can try this out with? The code looks good, just need to test it before I commit the code

**Comment by srijanshetty** [comment]
@nramadas just give me till the end of the week to fix some internal issues and then I'll share everything!

**Comment by srijanshetty** [comment]
@nramadas you can check for bounties on the URL: `http://localhost:3000/realm/ORCA/hub`.

We're good to go with the integration.

**Review by nramadas — APPROVED** [review]

**Pull request merged**